src/jobs/integration-test.yaml

```diff
@@ -0,0 +1,43 @@
+description: |
+  Runs an integration test by creating a KIND cluster and executing it as a Go
+  test.
+
+  Uses the machine executor and replaces the Go version with a version
+  controlled by us since the preinstalled Go is very outdated.
+
+  Tests can be customized by providing a KIND config file or a setup script
+  which is run before the test is executed.
+environment:
+  GOPATH: /go
+  KUBERNETES_VERSION: v1.16.3
```

Comment:
Where is this used?

Reply:
It's used here. https://github.com/giantswarm/architect-orb/blob/fcdc0e0f2834bfdc88a7534e2abbacd6211c1dc2/src/commands/integration-test-create-cluster.yaml#L22

IIRC I did this following a request from @MarcelMue that the k8s version be explicit and obvious.

Using an env var means you can override. I think this is useful for testing new k8s releases before we change the default.

Reply:
If we need to be able to override that then let's make this a parameter. I'd still like to avoid being able to do that because I'm afraid this can be abused. I can disagree and commit though.